.gitignore

```diff
@@ -29,3 +29,8 @@ dist
 
 # IntelliJ files
 .idea
+
+# SKY UX visual directory
```

Comment:
This file seems a bit minimal compared to our other libraries. Mind copy-pasting the `.gitignore` file from https://github.com/blackbaud/skyux-core/blob/master/.gitignore?